cedar-lean/Cedar/Thm/Data/List.lean

```diff
@@ -1056,8 +1412,7 @@ theorem find?_pair_map {α β γ} [BEq α] (f : β → γ) (xs : List (α × β)
   List.find? (λ x => x.fst == k) (List.map (λ x => (x.fst, f x.snd)) xs)
 := by
   induction xs
-  case nil =>
-    simp only [find?_nil, Option.map_none', map_nil]
+  case nil => simp
```

Comment:
These "bare simps" make me nervous.  Can we revert to `simp only`? :). Note that we use `simp only` elsewhere in this proof to close goals.

Reply:
Plenty of "bare simps" existing before this PR (including in this file), but I'll do a pass

Reply:
Noting that we have said previously that `simp` to close a goal (as here) should be perfectly stable

Reply:
Pushed a pass removing some bare simps

Reply:
There's a lot of other `simp` that don't have `simp only`, including in this file.  Can we not block this PR on this issue, and just make an issue to follow up?

Reply:
Definitely no need to block this PR on old bare `simp`s.  I should have been clearer:  my goal is to not introduce new ones and get rid of old ones eventually :)